Resources/public/templates/tabs/locations.hbt

```diff
@@ -25,8 +25,17 @@
                                     <a href="{{ path "viewLocation" id=id languageCode=contentInfo.mainLanguageCode }}">{{ pathString }}</a>
                                 </td>
                                 <td class="ez-table-data-childcount">{{ childCount }}</td>
-                                <td class="ez-table-data-visibility">
-                                    {{#if hidden}}Hidden{{else}}{{#if invisible}}Hidden by superior{{else}}Visible{{/if}}{{/if}}
+                                <td class="ez-table-data-visibility pure-g">
+                                    <div class="ez-table-data-visibility-text-box pure-u-1-2">
+                                        {{#if hidden}}Hidden{{else}}{{#if invisible}}Hidden by superior{{else}}Visible{{/if}}{{/if}}
+                                    </div>
+                                    <div class="ezpure-u-1-2">
+                                        <button
+                                            data-location-id="{{id}}"
+                                            class="ez-locations-hidden-button pure-button ez-font-icon ez-button">
+                                            {{#if hidden}}reveal{{else}}{{#if invisible}}{{else}}hide{{/if}}{{/if}}
```

Comment:
`Reveal` and `Hide`. And if the location is invisible, the button should also contain `Hide`